src/components/MultiHopTrade/components/FiatRamps/FiatRampTrade.tsx

```diff
@@ -198,18 +210,20 @@ const RampRoutes = memo(({ onChangeTab, direction }: RampRoutesProps) => {
     1000,
   )
 
-  useEffect(() => {
-    dispatch(tradeRampInput.actions.setSelectedFiatRampQuote(null))
-  }, [debouncedSellAmount, dispatch])
-
   useEffect(() => {
     dispatch(tradeRampInput.actions.setSelectedFiatRampQuote(null))
     dispatch(tradeRampInput.actions.setSellAmountCryptoPrecision('0'))
     dispatch(tradeRampInput.actions.setSellFiatAmount('0'))
   }, [direction, dispatch])
 
+  useEffect(() => {
+    dispatch(tradeRampInput.actions.setSelectedFiatRampQuote(null))
+    queryClient.resetQueries({ queryKey: ['rampQuote'] })
```

Comment:
q: Any reason why we `resetQueries` vs. `invalidateQueries` here?

Reply:
Tbh I think it was because of the keys missing from the query key, but seems fine now that I changed eveyrthing, changed back to invalidateQueries and updated the query keys in [2d2036c](https://github.com/shapeshift/web/pull/10636/commits/2d2036c48f648acac455b81885d608950b0bf9b2)